src/win32/kerberos_win32.cc

```diff
@@ -78,6 +78,10 @@ void KerberosClient::UnwrapData(const CallbackInfo& info) {
     });
 }
 
+static bool isStringTooLong(const std::string& str) {
+    return str.size() >= ULONG_MAX;
```

Comment:
I just noticed this field should be .length() to match the calls below, but let me know if the current change breaks styling guidelines. I wasn't sure whether to also put the throw statements in here.